dts/arm/st/l4/stm32l4s5.dtsi

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2020 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/l4/stm32l4r5.dtsi>
+
+/ {
```

Comment:
If stm32l4r5.dtsi is included, you shouldn't need all of this remaining stuff (this will be duplicated)

Reply:
ok, just including  <st/l4/stm32l4r5.dtsi>